config/prow/config.yaml

```diff
@@ -404,6 +404,7 @@ tide:
     labels:
     - lgtm
     - approved
+    - "cla: yes"
```

Comment:
Why does tide need to be made aware of the cla label?
If the CLA status check isn't passing then doesn't that prevent tide from merging the PR?

Your comment: https://github.com/kubernetes/test-infra/pull/18095
made it sound like the problem was the repo didn't have these labels defined which was causing the problem.

Reply:
Let me explain the context.

In https://github.com/kubeflow/pipelines/issues/1653

They mentioned when tide configuration doesn't exactly match github status. If an offending PR appears, tide bot will continuously try to merge the PR and fail. That will soon drain github api rate limit and affecting all other repos managed by tide.

Tide cannot read github statuses, so it must use a label to get statuses.

Reply:
Tide can check status contexts, and require them before merging, if you configure it to:
https://github.com/kubernetes/test-infra/blob/8c9d7c79573d0368573eea2ffd0403a8498f9b00/config/prow/config.yaml#L102-L104

This can be configured on a repo or org level.

Reply:
Yes, you can use either mechanism to make Tide respect that requirement as mentioned here: https://github.com/kubeflow/pipelines/issues/1653#issuecomment-650287176

Requiring the status context rather than the label is arguably better because if the google-bot posts a context and label that conflict, using the context as the source of truth will prevent Tide from clashing with the GH settings.

/ok-to-test

Reply:
Thanks @cblecker and @cjwagner!

I missed the later half of the comment. Then I think requiring a github status context is definitely better, will update this PR.